mldsa/src/sign.c

```diff
@@ -490,20 +490,36 @@ __contract__(
   unsigned int n;
   uint32_t z_invalid, w0_invalid, h_invalid;
   int ret;
+  typedef
+  /* TODO: Remove the following workaround for
+   * https://github.com/diffblue/cbmc/issues/8813 */
+#if defined(MLD_CONFIG_REDUCE_RAM)
```

Comment:
Nit: Wrap this into a `#define`, like `MLK_UNION_OR_STRUCT`.

Reply:
Okay. I also added a bit of commentary to explain the rationale.